dist/index.html

```diff
@@ -7,22 +7,20 @@
     <title>To-Do List</title>
 <script defer src="main.js"></script></head>
 <body>
-
-    <h1>Today's To-Do</h1>
-    <div class="loader"></div>
-
-
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/232189342-deca4c3a-f784-43cd-b9b6-5592dcfb6ad2.png)

**[OPTIONAL]**

- [ ] It would be nice if you make your commit messages more meaningful and capitalize 👍 because some of your commit messages are not capitalized and meaningful 😨 Please consider this issue to make your pull request professional and descriptive ☺️